src/Uno.UI/UI/Xaml/Controls/CalendarView/Primitives/CalendarViewTemplateSettings.cs

```diff
@@ -48,7 +48,7 @@ public string WeekDay1
 			"WeekDay1",
 			typeof(string),
 			typeof(CalendarViewTemplateSettings),
-			new FrameworkPropertyMetadata(default(string)));
+			new FrameworkPropertyMetadata(string.Empty));
```

Comment:
It seems like all these should be regular properties, not DPs. I'm going to leave this as string.Empty instead of adding `?? ""` since the DP doesn't actually exist in UWP.